packages/obojobo-document-engine/ObojoboDraft/Chunks/IFrame/viewer-component.js

```diff
@@ -216,6 +249,16 @@ export default class IFrame extends React.Component {
 								zoomReset={this.boundOnZoomReset}
 							/>
 						) : null}
+						{isShowing ? (
```

Comment:
We may want to move this up before the reload and zoom buttons; this would bracket the actual embedded content with the skip buttons.